strongbox-commons/pom.xml

```diff
@@ -65,6 +65,13 @@
     </build>
 
     <dependencies>
+
+        <dependency>
```

Comment:
@AlexOreshkevich,

Why does the `strongbox-commons` module need to depend on the `strongbox-data-service`?